src/ConnectedMode.UnitTests/Binding/UnintrusiveBindingPathProviderTests.cs

```diff
@@ -32,31 +30,30 @@ public class UnintrusiveBindingPathProviderTests
     {
         [TestMethod]
         public void MefCtor_CheckIsExported()
-        {
-            var serviceProvider = CreateConfiguredServiceProvider("");
```

Comment:
The old constructor called `serviceProvider.GetService`, so we had to provide it with a configured service provider. The new constructor doesn't.